lib/postgres_v3_experimental.dart

```diff
@@ -40,11 +40,23 @@ abstract class PgSession {
     Duration? timeout,
   });
 
+  /// Executes the [query] with the given [parameters].
+  ///
+  /// [query] must either be a [String] or a [PgSql] query with types for
+  /// parameters. When a [PgSql] query object is used, [parameters] can be a
+  /// list of direct values. Otherwise, it must be a list of [PgTypedParameter]s.
+  ///
+  /// When [schemaOnly] is set to true, the implementation may internally
```

Comment:
Why not name it `ignoreRows` instead? I don't know if one is better than the other, but the end result is mostly the absence of rows, isn't it?